### ISSUE TITLE
Docker Environment Variables for SMTP

### ISSUE THREAD

**Issue by mariushosting** [alextselegidis/easyappointments-docker#9]
Is it possible to include Environment Variables for the general SMTP configuration in Docker?
This makes configuration more practical and faster.

**Comment by alextselegidis** [comment]
Hello! 

Yes, that will be a helpful addition. 

Till this is done, you can volume your local /application/config/email.php file with the SMTP configuration 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>

**Comment by alextselegidis** [comment]
Quick update on this ticket: The modifications for env based SMTP configuration are almost completed and will soon become available to the public. 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>

**Comment by mariushosting** [comment]
> Quick update on this ticket: The modifications for env based SMTP configuration are almost completed and will soon become available to the public.
> 
> ![](https://camo.githubusercontent.com/b6ae4c709927e0f22bb9887f12b030ab995b6802b3d5fa2134f07bc4320a1a1c/68747470733a2f2f656173796170706f696e746d656e74732e6f72672f696d672f656173796170706f696e746d656e74732d7369676e61747572652e706e67)	Alex Tselegidis, Easy!Appointments Creator
> [Need a customization? Get a free quote!](https://easyappointments.org/get-a-free-quote)

Awesome! Thank you for the update!

**Comment by chris2deuce** [comment]
Hi. Can't wait until docker v1.5.1 is available. :-)
@alextselegidis Could you also create a beta version for the docker container?

**Comment by valterbarreira** [comment]
How do i set up email configuration with docker?

**Comment by alextselegidis** [comment]
Hello! 

The updated docker image will allow users to set email confirmation via env variables. 

It will be released in the next few days 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>

**Comment by omergencoglu** [comment]
Hi @alextselegidis , thank you for your great work! I'm looking forward to the Docker v1.5.1. Do you have any updates?

**Comment by alextselegidis** [comment]
Hello! 

Yes, I'm running some tests currently and after they are completed I will release the 1.5.1 image on Docker Hub. 

Star and follow the project to get updates on new releases. 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>

**Comment by valterbarreira** [comment]
Hi, have you already released the new image?

**Comment by alextselegidis** [comment]
Problem fixed and released on docker hub 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>

**Comment by mariushosting** [comment]
> Problem fixed and released on docker hub
> 
> ![](https://camo.githubusercontent.com/b6ae4c709927e0f22bb9887f12b030ab995b6802b3d5fa2134f07bc4320a1a1c/68747470733a2f2f656173796170706f696e746d656e74732e6f72672f696d672f656173796170706f696e746d656e74732d7369676e61747572652e706e67)	Alex Tselegidis, Easy!Appointments Creator
> [Need a customization? Get a free quote!](https://easyappointments.org/get-a-free-quote)

Thank You! They work like a charm!

**Comment by valterbarreira** [comment]
> > Problem fixed and released on docker hub
> > ![](https://camo.githubusercontent.com/b6ae4c709927e0f22bb9887f12b030ab995b6802b3d5fa2134f07bc4320a1a1c/68747470733a2f2f656173796170706f696e746d656e74732e6f72672f696d672f656173796170706f696e746d656e74732d7369676e61747572652e706e67)	Alex Tselegidis, Easy!Appointments Creator
> > [Need a customization? Get a free quote!](https://easyappointments.org/get-a-free-quote)
> 
> Thank You! They work like a charm!

Where do i set smtp variables? On docker-compose.yml?

**Comment by mariushosting** [comment]
> > > Problem fixed and released on docker hub
> > > ![](https://camo.githubusercontent.com/b6ae4c709927e0f22bb9887f12b030ab995b6802b3d5fa2134f07bc4320a1a1c/68747470733a2f2f656173796170706f696e746d656e74732e6f72672f696d672f656173796170706f696e746d656e74732d7369676e61747572652e706e67)	Alex Tselegidis, Easy!Appointments Creator
> > > [Need a customization? Get a free quote!](https://easyappointments.org/get-a-free-quote)
> > 
> > 
> > Thank You! They work like a charm!
> 
> Where do i set smtp variables? On docker-compose.yml?

Yes. You can follow my compose here: https://mariushosting.com/how-to-install-easy-appointments-on-your-synology-nas/
